.github/workflows/fetch-and-ingest-genbank-master.yml

```diff
@@ -4,23 +4,16 @@ on:
   schedule:
     # Note times are in UTC, which is 1 or 2 hours behind CET depending on daylight savings.
     #
-    # Currently, we aim to trigger ingest every day at 18:07 UTC which is 19:07 CET (as of Mar 2022).
-    # Note the actual runs might be late. As of right now, the action starts around 20 past the hour.
-    # Numerous people were confused, about that, including me:
-    #  - https://github.community/t/scheduled-action-running-consistently-late/138025/11
-    #  - https://github.com/github/docs/issues/3059
+    # Currently, we aim to trigger runs at 6pm UTC on Saturday
     #
     # Note, '*' is a special character in YAML, so you have to quote this string.
     #
     # Docs:
     #  - https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#schedule
     #
-    # Tool that deciphers this particular format of crontab string:
-    #  - https://crontab.guru/
-    #
```

Comment:
I go to this website whenever I want to understand GitHub Actions schedule syntax, so the link is useful. However, I do see that it's linked in the GitHub doc link above, so I'm fine with the removal.